app/konnect/mesh-manager/rbac.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Mesh Manager RBAC
+content_type: reference
+---
+
+Mesh Manager has it's own role-based access control (RBAC) settings that are separate from the {{site.konnect_short_name}} RBAC settings. The Mesh Manager RBAC settings are specific to the meshes and mesh policies in Mesh Manager. 
+
+To completely configure RBAC for Mesh Manager, you must configure both roles and role bindings.
+* **Role:** Determines what resources a user or group has access to
+* **Role binding:** Determines which users or groups are assigned to a particular role
+
+<!-- Is the admin role included by default/automatically? -->
+
+## Roles description
+
+You can use the following example roles as guides when you are configuring RBAC for Mesh Manager:
+
+| Role                        | Description  |
+|-----------------------------|--------------|
+| Admin | The admin is the service mesh operator and they are a part of infrastructure team responsible for {{site.mesh_product_name}} deployment. |
+| Service owner | The service owner is a part of the team that is responsible for a given service. |
+| Observability operator | The observability operator is a member of the infrastructure team that is responsible for the logging, metrics, and tracing systems in the organization. Currently, those features are configured on `Mesh`, `TrafficLog`, and `TrafficTrace` objects. |
+| Single mesh operator | With {{site.mesh_product_name}}, you can segment the deployment into many logical service meshes configured by the `Mesh` object. You can add a role that grants access to one specific mesh and all objects connected with this mesh instead of all meshes. |
+
+## Key mapping
+
+### Roles
```

Comment:
I think this section can also be deleted.

Reply:
Actually I think maybe we can just provide a high level description of roles, maybe stolen from the main RBAC page (and also link to more information about creating / crafting roles by linking to the main Kong Mesh RBAC page). Something like:

```
AccessRoles specify kinds of access and resources to which access is granted. Note that access is defined only for write operations. Read access is available to all users who have the Konnect Mesh Global Control Plane `viewer` role. AccessRoles define roles that are assigned separately to users and groups / teams via AccessRoleBindings. They are global-scoped, which means they are not bound to a mesh. 
```

We might want to hyperlink the Konnect Mesh Global Control Plane viewer role reference above to the page describing Konnect-wide roles that we currently have changes open for in the main MinK PR?